react/Button/ButtonLink.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import cx from 'classnames'
+import PropTypes from 'prop-types'
+import styles from './styles'
+
+const ButtonLink = props => {
+  const { theme, href, target, className, children, onClick } = props
+  return (
+    <a
+      href={href}
+      target={target ? '_blank' : undefined}
+      className={cx(
+        styles['c-btn'], {
+          [styles[`c-btn--${theme}`]] : theme
+        },
+        className)}
+      onClick={onClick}
+    >
+      {children}
+    </a>
+  )
+}
+
+ButtonLink.propTypes = {
+  children: PropTypes.node.isRequired,
+  theme: PropTypes.string,
+  href: PropTypes.string.isRequired,
+  target: PropTypes.bool,
+  className: PropTypes.string,
+  onClick: PropTypes.func
+}
+
+ButtonLink.defaultProps = {
+  theme: '',
+  href: '',
+  target: false,
+  className: '',
+  onClick: null
+}
+
+export default ButtonLink
```

Comment:
I think `Button` and `ButtonLink` should be in the same file since they share so much.

For example the className function can be shared.

```
const btnClass = function (theme, className) {
  return cx(
        styles['c-btn'], {
          [styles[`c-btn--${theme}`]] : theme
        },
        className)
}
```

```
className={ bntClass(theme, className) }
```

Some PropTypes and defaultPropTypes could be shared too. Since proptypes and defaultProps are regular javascript object, you can use spreading to share attributes.

Example : 

```
const btnPropTypes = {
  children: PropTypes.node.isRequired,
  theme: PropTypes.string,
  className: PropTypes.string,
  onClick: PropTypes.func
}

Button.PropTypes = {
   ...btnPropTypes,
  busy: PropTypes.bool
}

BtnLink.PropTypes = {
   ...btnPropTypes,
  href: PropTypes.string.isRequired,
  target: PropTypes.bool,
}
```

I think it would make the code more maintainable since there would be less risk of divergence between the two files.

Reply:
Well the buttons were in the same file but something breaks with Styleguidist, something related to the fact that those components are not exported as default. I don't really get it because the error i got is `String contains an invalid character` which is not really explicit.
I can try again though, maybe with a clear mind now…